grade.go

```diff
@@ -401,40 +417,26 @@ func main() {
 
 		result.StudentId = hydratedStudent.Id
 
-		if language == "python3" || language == "python" {
+		if language == "python3" || language == "python" || language == "javascript" || language == "js" {
 			result.CompileSuccess = true
 
-			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, input)
-			fmt.Printf("Output for %s: %s", result.Student, stdout)
-			result.RunCorrect, result.Feedback = compare(expected, stdout)
+			stdout := runInterpreted(filepath.Join(workDir, dir), runArgs, language, input)
+			fmt.Printf("Output For %s: %s", result.Student, stdout)
+			result.RunCorrect, result.Feedback = processOutput(expected, stdout)
+
+			// if student doesn't exist, commit to db
```

Comment:
another stray comment, could be worth another check for misplaced lines

Reply:
Going to review to make sure

Reply:
Reviewed and checked all of grade.go, which was the only file I really changed a lot during the rebase. I compared where comments used to go based off of previous commit history and have now moved them to the correct spot.